pkg/asset/installconfig/aws/permissions.go

```diff
@@ -324,6 +327,11 @@ var permissions = map[PermissionGroup][]string{
 		// Needed to filter zones by instance type
 		"ec2:DescribeInstanceTypeOfferings",
 	},
+	PermissionCreateEdgeCompute: {
+		"ec2:DescribeCarrierGateways",
+		"ec2:CreateCarrierGateway",
+		"ec2:DeleteCarrierGateway",
```

Comment:
I'll add a comment here that these permissions are used by CAPA when provisioning the edge infra.